### PR TITLE
plan section completed

### PR DESCRIPTION
**Pull request by FPurichaya**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -248,8 +248,8 @@ <h2>Our customers can't live without us</h2>
                 <!--canbe row or col-->
                 <blockquote>
                     <!--put quote-->
-                    "Omnifood is just awesome! I just launched a startup which leaves me with no time for cooking, so
-                    Omnifood is a life-saver. Now that I got used to it, I couldn't live without my daily meals!"
+                    Omnifood is just awesome! I just launched a startup which leaves me with no time for cooking, so
+                    Omnifood is a life-saver. Now that I got used to it, I couldn't live without my daily meals!
```
Comment (pawelfraczyk) [inline index.html]: this is a comment of reviewer

File: index.html [hunk 2]
```diff
@@ -268,14 +268,84 @@ <h2>Our customers can't live without us</h2>
...
+            <div class="col span-1-of-3">
+                <div class="planBox">
+                    <div>
+                        <h3>Premium</h3>
+                        <p class="planPrice">$399 <span>/ month</span></p> <!--span is to appear inside the paragraph, and style some text in the different way than the rest-->
+                        <p class="planPriceMeal">That’s only 13.30$ per meal</p>
+                    </div>
+                    <div>
+                        <ul>
+                            <li><i class="icon ion-md-checkmark citiesIcon"></i>1 meal every day</li>
```
Comment (pawelfraczyk) [inline index.html]: another comment of review

File: index.html (hunk 1 above)
Comment (FPurichaya) [reply index.html]: done

File: index.html (hunk 2 above)
Comment (FPurichaya) [reply index.html]: done

**Comment by FPurichaya** [comment]
plan section

**Pull request merged**